bin/_blackbox_common.sh

```diff
@@ -164,6 +164,19 @@ function add_filename_to_cryptlist() {
   fi
 }
 
+# Removes a file from the list of encrypted files
+function remove_filename_from_cryptlist() {
+  # If the name is not already on the list, this is a no-op.
+  local name=$(vcs_relative_path "$1")
+
+  if ! grep -s -q "$name" "$BB_FILES" ; then
```

Comment:
Suggestion: add -xF to grep so that it matches entire lines that are exact matches (no regex).